modules/nf-core/samtools/calmd/main.nf

```diff
@@ -0,0 +1,34 @@
+
+process SAMTOOLS_CALMD {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
```suggestion
    label 'process_medium'
```
Would this make more sense as a medium since it has a task.cpus? I don't know how this scales.